src/Services/Catalog/Catalog.Application/Common/Jobs/IUpdateStockCountJob.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Catalog.Application.Common.Jobs;
+
+public interface IUpdateStockCountJob
+{
+    public Task Execute();
```

Comment:
Add Async

Reply:
you can't add async to methods with no body